docs/pages/admin-guides/management/guides/aws-iam-identity-center.mdx

```diff
@@ -0,0 +1,285 @@
+---
```

Comment:
We need to pin the mdx file in our config.json https://github.com/gravitational/teleport/blob/master/docs/config.json#L2284

Reply:
Yeah, navigation is generated automatically now.